tests/python/test_graph.py

```diff
@@ -130,6 +129,12 @@ def test_plan_memory():
     y = sym.add(y, y)
     g = graph.create(y)
     g._set_json_attr("shape_attr_key", "shape")
+    reused_bytes = 16
```

Comment:
Maybe it's not clear. This is a test case where in the shared pool, there're 16 bytes on device 0, another 1 byte on device 0, and 32 bytes on device 9. Without the shared pool it takes 64 bytes. With this shared pool, total allocated mem should be 64 + 32 = 96 bytes for this graph. Should I test in finer granularity??